pallets/runtime/common/src/bridge.rs

```diff
@@ -243,6 +244,8 @@ decl_storage! {
         /// authorizations and are able to get their proposals delivered. The bridge creator
         /// transfers some POLY to their identity.
         Controller get(fn controller) build(|config: &GenesisConfig<T>| {
+            use frame_support::debug;
```

Comment:
This can go to the main `use` list in the preamble.

Reply:
It throws a warning when it is moved to the main. I guess that is due to the code generated by macro expansion.